docs/groovy-script/groovy/operators.md

```diff
@@ -0,0 +1,240 @@
+---
+order: 750
+---
+
+# Operators
+
+
+Groovy has many operators.
+Some are reused for different purposes in different contexts.
+In particular, many are used for [operator overloading](#operator-overloading), allowing the operator to be used on valid objects.
+
+You can read more about operators on the [official documentation](https://docs.groovy-lang.org/latest/html/documentation/core-operators.html).
+
+
+## Types of Operators
+
+
+### Arithmetic
+
+
+The basic math operators are the same as they always are.
+
+| Symbol   | Name           | Operation                                |
+|----------|----------------|------------------------------------------|
+| `a + b`  | Addition       | add `a` and `b`                          |
+| `a - b`  | Subtraction    | subtract `b` from `a`                    |
+| `a / b`  | Division       | divide `a` by `b`                        |
+| `a * b`  | Multiplication | multiple `a` by `b`                      |
+| `a % b`  | Remainder      | get the remainder of dividing `a` by `b` |
+| `a ** b` | Power          | `a` to the power of `b`                  |
+
+
+### Compound Assignment
+
+
+You can use several different operators in conjunction with an equal sign to update a variable.
+This will also implicitly cast the output into the same type as `a`.
+
+`a [op]= b` is equivalent to `a = a [op] b`.
+
+| Symbol     | Name                 | Operation     |
+|------------|----------------------|---------------|
+| `a = b`    | Assignment           | `a = b`       |
+| `a =+ b`   | Addition             | `a = a + b`   |
+| `a =- b`   | Subtraction          | `a = a - b`   |
+| `a =/ b`   | Division             | `a = a / b`   |
+| `a =* b`   | Multiplication       | `a = a * b`   |
+| `a =% b`   | Remainder            | `a = a % b`   |
+| `a =** b`  | Power                | `a = a ** b`  |
+| `a ?= b`   | Elvis                | `a = a ?: b`  |
+| `a &= b`   | Bitwise And          | `a = a & b`   |
+| `a \|= b`  | Bitwise OR           | `a = a \| b`  |
+| `a <<= b`  | Left Shift           | `a = a << b`  |
+| `a >>= b`  | Right Shift          | `a = a >> b`  |
+| `a >>>= b` | Right Shift Unsigned | `a = a >>> b` |
```

Comment:
Its `+=` not `=+` afaik. Does both work in groovy? Does that apply to all these operators?

Reply:
that is my mistake, it should definitely be `[op]=`